v2/internal/frontend/desktop/windows/frontend.go

```diff
@@ -709,14 +712,13 @@ func (f *Frontend) processMessageWithAdditionalObjects(message string, sender *e
 			return
 		}
 
-		_count, err := objs.GetCount()
+		count, err := objs.GetCount()
 		if err != nil {
 			f.logger.Error(err.Error())
 			return
 		}
 
 		files := []File{}
-		count := *(*uint32)(unsafe.Pointer(&_count))
 		for i := uint32(0); i < count; i++ {
 			_file, err := objs.GetValueAtIndex(i)
```

Comment:
Maybe we also need to Release here? because we will return if there is an error in the next line.

Reply:
Using `defer` is a common technique to make sure it's released no matter how the function exists. The defer could release all the memory allocated

Reply:
Thanks, I have updated the code

Reply:
I have a sneaky suspicion that this will only release the last _file. I think it will also attempt a release if `err != nil`, which might fail spectacularly as `_file` is unlikely to have been allocated.